src/docfx/config/ops/OpsConfig.cs

```diff
@@ -29,5 +29,9 @@ internal class OpsConfig
         [JsonProperty(nameof(ECMA2Yaml))]
         [JsonConverter(typeof(OneOrManyConverter))]
         public ECMA2YamlRepoConfig[]? ECMA2Yaml { get; private set; }
+
+        [JsonProperty("monikerPath")]
+        [JsonConverter(typeof(OneOrManyConverter))]
+        public string[]? MonikerPath { get; set; }
```

Comment:
>MonikerPath [](start = 25, length = 11)

could other reference language also uses this configuration? the name seems too vague to be associated with PowerShell

Reply:
If there's a name collision, we may:
- either ask plugin team to come up with a new name
- consume the plugin name in `customized_tasks` to distinguish, which I wouldn't propose :/
Simply change the `Property` name won't change anything